pkg/rpc/agent_redis_request_reply.go

```diff
@@ -46,8 +47,10 @@ type AgentInfo struct {
 func (a *AgentClient) Ping(ctx context.Context) ([]AgentInfo, error) {
 	replyChanName := xid.New().String()
 	ps := a.client.Subscribe(ctx, replyChanName)
-	defer ps.Close()
-	defer ps.Unsubscribe(context.Background(), replyChanName)
+	defer func() {
+		_ = ps.Close()
+		_ = ps.Unsubscribe(context.Background(), replyChanName)
+	}()
```

Comment:
is this change required by linter or it is for readability?

Reply:
It is required by linter, as all ignored return values should be discarded explicitly.